index.html

```diff
@@ -218,18 +214,18 @@ <h3 id="matters-considered-by-the-placement-rules">Considerations for  simple pl
     In the first step, processing of layout only considers  the relative positions of the ruby 
     annotation and its base text. 
     In the second step,  layout processing decides the position of the <a title="ruby block">ruby block</a> in the line, taking into consideration the preceding and 
-    following characters. On the other hand, relative positions of the ruby annotation and its base text as decided in the first step are not modified 
-    in the light of any characters preceding and following the ruby block. 
+    following characters. Note that relative positions of the ruby annotation and its base text as decided in the first step are not modified 
```

Comment:
Since the original Japanese text is 逆に言えば, IMHO "in other words" is more appropriate than "on the other hand" or "note that".

I also suggested this in https://github.com/w3c/simple-ruby/pull/17#discussion_r423505639

Reply:
Fixed in 2nd commit.